lib/EnsEMBL/REST/Model/LDFeatureContainer.pm

```diff
@@ -99,23 +94,18 @@ sub fetch_LDFeatureContainer_slice {
     $ldfca->db->vcf_root_dir($ld_config->{dir}) if (defined $ld_config->{dir});
   }
 
-  my $population_name = $c->request->param('population_name');
-  if ($population_name) {
-    my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');     
-    my $population = $pa->fetch_by_name($population_name);
-    if (!$population) {
-      Catalyst::Exception->throw("Could not fetch population object for population name: $population_name");
-    }
-    my $ldfc;
-    try {
-      $ldfc = $ldfca->fetch_by_Slice($slice, $population);
-    } catch {
-      Catalyst::Exception->throw("Something went wrong while fetching from LDFeatureContainerAdaptor");
-    };
-    return $self->to_array($ldfc)
+  my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');     
+  my $population = $pa->fetch_by_name($population_name);
+  if (!$population) {
+    Catalyst::Exception->throw("Could not fetch population object for population name: $population_name");
   }
-  my $ldfc = $ldfca->fetch_by_Slice($slice);
-  return $self->to_array($ldfc);
+  my $ldfc;
+  try {
+    $ldfc = $ldfca->fetch_by_Slice($slice, $population);
+  } catch {
+    Catalyst::Exception->throw("Something went wrong while fetching from LDFeatureContainerAdaptor");
```

Comment:
Does the ldfca error make it to the log files, or have you swallowed it here on purpose?